package.json

```diff
@@ -12,14 +12,17 @@
   },
   "homepage": "https://thelounge.github.io/",
   "scripts": {
-    "coverage": "nyc mocha",
+    "coverage": "rm -rf .nyc_output/ && npm-run-all coverage:* && cp test/.nycrc-report .nycrc && nyc report json && rm .nycrc",
+    "coverage:mocha": "cp test/.nycrc-mocha .nycrc && nyc npm run test:mocha && rm .nycrc",
+    "coverage:mocha-webpack": "cp test/.nycrc-mocha-webpack .nycrc && nyc npm run test:mocha-webpack && rm .nycrc",
```

Comment:
Please ignore the stupid `cp`/`rm` dance for now. It will be removed soon. This is because at the moment, `.nycrc` path cannot be custom. Thankfully, this is just about to change: https://github.com/istanbuljs/nyc/pull/724